tests/test_mark_tokens.py

```diff
@@ -550,3 +561,21 @@ def test_complex_slice_and_parens(self):
     source = 'f((x)[:, 0])'
     m = self.create_mark_checker(source)
     m.verify_all_nodes(self)
+
+  def test_sys_modules(self):
+    for module in list(sys.modules.values()):
+      try:
+        filename = inspect.getsourcefile(module)
+      except TypeError:
+        continue
+
+      if not filename:
+        continue
+
+      filename = os.path.abspath(filename)
+      print(filename)
```

Comment:
print calls probably don't belong in a test case

Reply:
Maybe, but it's very useful in development and even when tests pass it's quite nice to see all the files that were tested. This is what I've done in my own library.